tools/serverpod_cli/lib/src/database/migration.dart

```diff
@@ -43,15 +43,18 @@ DatabaseMigration generateDatabaseMigration({
 
   // Find added or modified tables
   for (var dstTable in targetTables) {
-    var srcTable = sourceTables.cast<TableDefinition?>().firstWhere(
+    var srcTable = databaseSource.tables.cast<TableDefinition?>().firstWhere(
```

Comment:
Question: Now that we don't use `sourceTables` here we don't benefit from the filtering on line 16 (removing the serverpod_migrations table). Does that need to be reflected here as well?

Reply:
I can actually see that we add additional `serverpod_migration` tables for our modules because this filtering is missing in the next commit.

Reply:
Since we are looking for a specific named table we should never encounter the serverpod_migration table here as it has been removed from the targetTables.

Reply:
Aah you are correct! 👍